usaspending_api/download/helpers/download_file_helpers.py

```diff
@@ -27,10 +27,10 @@ def get_last_modified_download_file_by_prefix(download_prefix: str) -> Optional[
         s3_resource = boto3_session.resource(
             service_name="s3", region_name=CONFIG.AWS_REGION, endpoint_url=f"http://{CONFIG.AWS_S3_ENDPOINT}"
         )
-        s3_bucket = s3_resource.Bucket(settings.BULK_DOWNLOAD_S3_BUCKET_NAME)
+        s3_bucket = s3_resource.Bucket(settings.DATABASE_DOWNLOAD_S3_BUCKET_NAME)
```

Comment:
I believe changing this bucket name will break the unlinked award download. Perhaps the bucket name should be passed into this function as an additional parameter. That might warrant a function rename as well then. Could potentially just drop the `_file_by_prefix` prefix.